src/main/java/filter/expression/QualifierContent.java

```diff
@@ -0,0 +1,66 @@
+package filter.expression;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+public class QualifierContent {
+
+    //Only one of these will exist in a QualifierContent object
```

Comment:
Hmm, true. The intention was for us to be able to treat the output of these parsing functions in a polymorphic way. If we can do this using just qualifiers, that would be ideal.